templates/includes/sidebar.html

```diff
@@ -0,0 +1,13 @@
+<ul class="sidebar-nav">
+  <li>
+    <a href="#home-page">Home page</a>
+  </li>
+  <li>
+    <a href="#manufacturers">Manufacturers</a>
+  </li>
+  <li>
+    <a href="#cars">Cars</a>
+  </li>
+  <li>
+    <a href="#drivers">Drivers</a>

```

Comment:
Replace the `#` in the `href` attributes with actual URL paths for each section (e.g., `{% url 'index' %}` for the home page). This will make the navigation links functional.